src/system.py

```diff
@@ -3372,7 +3697,34 @@ class GroundNodes(Enum):
             'ADCS_SET_STAR_TRACK_CONFIG': {
                 'subPort': 97,
                 'inoutInfo': {
-                    'args': ['>f4', '>f4', '>f4', '>u2', '>u2', '>u1', '>u1', '>u1', '>u2', '>u1', '>u1', '>u1', '>f4', '>f4', '>f4', '>f4', '>f4', '>f4', '>f4', '>u1', '>u1', '>u1', '>?', '>?', '>u1'],
+                    'args': { #TODO Better names
```

Comment:
I'm going to ask the ADCS to fix the rest of the names. Doesn't need to be part of this PR